packages/pluggableWidgets/color-picker-web/src/ColorPicker.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget
+    id="com.mendix.widget.custom.colorpicker.ColorPicker"
+    needsEntityContext="true"
+    pluginWidget="true"
+    offlineCapable="true"
+    xmlns="http://www.mendix.com/widget/1.0/"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../node_modules/mendix/custom_widget.xsd"
+>
+    <name>Color picker</name>
+    <description>Pick a color from color input</description>
+    <studioProCategory>Display</studioProCategory>
+    <studioCategory>Display</studioCategory>
+    <properties>
+        <propertyGroup caption="General">
+            <propertyGroup caption="Data source">
+                <property key="colorAttribute" type="attribute">
```

Comment:
Does it make sense to make this property required?

Reply:
As I understand it required by default if you have `needsEntityContext="true"` in `widget` attributes.
Because now it's giving error in studio pro when you didn't select entity

Reply:
TIL. Should be fine then.